.github/workflows/ci.yml

```diff
@@ -204,14 +204,14 @@ jobs:
         if: ${{ success() || failure() }}
         run: |
           yarn dev:backend 2>&1 | tee var/logs/backend.log & ## ampersand enables background mode
-          yarn wait-on --timeout 30000 http://localhost:5001
+          yarn wait-on --timeout 30000 http://0.0.0.0:5001
 
       - name: Build and launch frontend
         if: ${{ success() || failure() }}
         run: |
           yarn workspace @hashintel/hash-frontend next build
           yarn workspace @hashintel/hash-frontend start 2>&1 | tee var/logs/frontend.log & ## ampersand enables background mode
-          yarn wait-on --timeout 30000 http://localhost:3000
+          yarn wait-on --timeout 30000 http://0.0.0.0:3000
```

Comment:
Using `localhost:3000` produced this error in CI:

```
yarn run v1.22.18
$ next start
ready - started server on 0.0.0.0:3000, url: http://localhost:3000/
Error: Timed out waiting for: http://localhost:3000/
```

Next app was running, but `wait-on` could not ‘see’ it. I was able to reproduce the same behavior locally on Node 18. Here’s the output of `yarn wait-on --timeout 30000 http://localhost:3000 --verbose`:

```
making HTTP(S) head request to  url:http://localhost:3000 ...
  HTTP(S) error for http://localhost:3000 Error: connect ECONNREFUSED ::1:3000
```

Not sure why, but replacing `localhost` with `0.0.0.0` helped. We can keep this hack for now and bring back `localhost` again after upgrading more dependencies. Upgrading `wait-on` to 6.0.1 did not help; upgrading `next` broke `yarn workspace @hashintel/hash-frontend build`:

```
info  - Skipping validation of types  
info  - Creating an optimized production build  
Failed to compile.

../../../node_modules/@sentry/cli/js/helper.js:3:0
Module not found: Can't resolve 'child_process'

Import trace for requested module:
../../../node_modules/@sentry/cli/js/index.js
../../../node_modules/@sentry/nextjs/esm/config/webpack.js
../../../node_modules/@sentry/nextjs/esm/config/index.js
../../../node_modules/@sentry/nextjs/esm/index.server.js
./src/components/sandbox/FramedBlock/FramedBlock.tsx
./src/components/sandbox/FramedBlock/index.tsx

https://nextjs.org/docs/messages/module-not-found

../../../node_modules/@sentry/nextjs/esm/config/webpack.js:6:0
Module not found: Can't resolve 'fs'
```

We can address this in a separate PR.